tests/kernel/mutex/mutex/src/mutex.c

```diff
@@ -428,3 +422,14 @@ K_THREAD_ACCESS_GRANT(THREAD_07, &mutex_3);
 K_THREAD_ACCESS_GRANT(THREAD_08, &mutex_2);
 K_THREAD_ACCESS_GRANT(THREAD_09, &mutex_1);
 K_THREAD_ACCESS_GRANT(THREAD_11, &mutex_3);
+
+/*test case main entry*/
+void test_main(void)
+{
+	k_thread_access_grant(k_current_get(), &private_mutex,
+			      &mutex_1, &mutex_2, &mutex_3, &mutex_4,
+			      &thread_12_thread_data, &thread_12_stack_area,
+			      NULL);
+	ztest_test_suite(test_mutex, ztest_unit_test(testing_mutex));
```

Comment:
can you use ztest_user_unit_test() instead of dropping to user mode later?

Reply:
Thanks that helped.